yass-components-react/src/components/Grid/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactNode } from "react";
+
+import * as classNames from "../../../types/yass.json";
+
+export interface GridProps {
+  id?: string;
+  testId?: string;
+  children?: ReactNode;
+  gridTemplateColumns: typeof classNames["grid-template-columns"][number];
+  gap?: typeof classNames["gap"][number];
+  columnGap?: typeof classNames["column-gap"][number];
+  rowGap?: typeof classNames["row-gap"][number];
+  justifyContent?: typeof classNames["justify-content"][number];
+}
+
+const Grid = ({
+  id,
+  testId,
+  children,
+  gridTemplateColumns = "grid-template-columns:three-column-layout",
```

Comment:
I'm low key chuffed with how this turned out. Being able to define custom column layouts as tokens and then set them on `grid-template-columns` :chefkiss: